packages/react-components/react-carousel-preview/library/src/components/Carousel/Carousel.types.ts

```diff
@@ -33,6 +33,17 @@ export type CarouselProps = ComponentProps<CarouselSlots> & {
    * Circular enables the carousel to loop back around on navigation past trailing index.
    */
   circular?: boolean;
+
+  /**
+   * Group size enables pagination of multiple cards as a single slide
+   */
+  groupSize?: number;
```

Comment:
@default: 1?

Reply:
do we want to try and handle cases where a user (likely accidentally) inputs non positive or non-whole numbers?